examples/notebooks/b - Secure Classification with TFE Keras - Secure Model Serving .ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>Sorry one last thing! Shouldn't `num_steps=3` here?  Otherwise the last cell will hang until it serves two more prediction requests.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
either that or num_tests in the other notebook should be 5

Reply:
ah good catch, let me fix this.